x-pack/plugins/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart.js

```diff
@@ -982,11 +984,14 @@ class TimeseriesChartIntl extends Component {
       .domain(this.calculateContextXAxisDomain());
 
     const combinedData =
-      contextForecastData === undefined ? data : data.concat(contextForecastData);
+      showForecast && Array.isArray(contextForecastData) ? data.concat(contextForecastData) : data;
+
     const valuesRange = { min: Number.MAX_VALUE, max: Number.MIN_VALUE };
     each(combinedData, (item) => {
-      valuesRange.min = Math.min(item.value, valuesRange.min);
-      valuesRange.max = Math.max(item.value, valuesRange.max);
+      const lowerBound = item.lower ?? Number.MIN_VALUE;
```

Comment:
I think these are the wrong way round, and should be 

```
      const lowerBound = item.lower ?? Number.MAX_VALUE;
      const upperBound = item.upper ?? Number.MIN_VALUE;
```

Otherwise for the case where there is no model plot you see this with your change:
![image](https://user-images.githubusercontent.com/7405507/115691919-657ae580-a356-11eb-9c3e-440004f7cefa.png)

Before your edit:
<img width="1551" alt="smv_context_1" src="https://user-images.githubusercontent.com/7405507/115691711-35334700-a356-11eb-8c39-13b602cd907e.PNG">